lib/engines/hckinstall/hckinstall.rb

```diff
@@ -15,23 +15,205 @@ class HCKInstall
 
     attr_reader :platform
 
+    PLATFORMS_JSON = 'lib/engines/hcktest/platforms.json'
+    ISO_JSON = 'lib/engines/hckinstall/iso.json'
+
+    ANSWER_FILES = ['autounattend.xml', 'unattend.xml'].freeze
+    CLIENTS = %w[c1 c2].freeze
+
+    STUDIO_INSTALL_TIMEOUT = 3 * 60 * 60 # 3 hour
```

Comment:
I think moving those parameters to JSON file is a better idea.

Reply:
Will do this